includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -258,6 +261,29 @@ public function decrease_usage_count( &$coupon, $used_by = '' ) {
 				$coupon->set_used_by( (array) get_post_meta( $coupon->get_id(), '_used_by' ) );
 			}
 		}
+		return $new_count;
+	}
+
+	/**
+	 * Increase or decrease the usage count for a coupon by 1.
+	 *
+	 * @since 2.7.0
+	 * @param WC_Coupon
+	 * @param string $operation 'increase' or 'decrease'
+	 * @return int New usage count
+	 */
+	private function vary_usage_count_meta( &$coupon, $operation = 'increase' ) {
```

Comment:
I got it from [here](http://english.stackexchange.com/questions/55416/hypernym-for-increase-and-decrease) :) Update is good, too. I'll switch it over.